Sources/RibbonKit/List/RibbonListSectionConfiguration.swift

```diff
@@ -10,21 +10,24 @@ public struct RibbonListSectionConfiguration: Hashable {
     public var sectionInsets: UIEdgeInsets
     public var headerInsets: UIEdgeInsets
     public var footerInsets: UIEdgeInsets
+    public var listConfiguration: RibbonListCollectionViewListConfiguration
 
     public init(
         layout: RibbonListSectionLayout,
         interItemSpacing: CGFloat = 6.0,
         interGroupSpacing: CGFloat = 6.0,
         sectionInsets: UIEdgeInsets = .zero,
         headerInsets: UIEdgeInsets = .zero,
-        footerInsets: UIEdgeInsets = .zero
+        footerInsets: UIEdgeInsets = .zero,
+        listConfiguration: RibbonListCollectionViewListConfiguration = .init(appearance: .plain)
```

Comment:
Is it necessary to have an additional property here? Because this property is only necessary if `layout == .list`, otherwise it's useless. I would add it as associated value to the `RibbonListSectionLayout.list(RibbonListCollectionViewListConfiguration)`.

Reply:
Code updated, what do you think?